[MOV] accounting: restructuration of the accounting documentation (#4422)
odoo/documentation: content/applications/finance/accounting.rst

accounts, smart matching suggestions, etc.

.. seealso::
   `Odoo Tutorials: Accounting <https://www.odoo.com/slides/accounting-19>`_

.. cards::

   .. card:: Getting started
      :target: accounting/getting_started
      :large:

>       Basic concepts of accounting and initial setup of your accounting

Reviewer: ```suggestion
      Basic concepts of accounting and initial setup of accounting
```
remove all "you"/"your" mentions in the doc

   .. card:: Accounts receivable
      :target: accounting/receivables

      Customer invoices, payment terms, and electronic invoicing

   .. card:: Accounts payable
      :target: accounting/payables

      Vendor bills, assets, and invoice digitization (OCR)

   .. card:: Payments
      :target: accounting/payments

>       Invoices and bills payments (online, checks, batches) and follow-up on invoices

Reviewer: ```suggestion
      Invoice/bill payments (online, checks, batches) and follow-up on invoices
```

   .. card:: Bank and cash accounts
      :target: accounting/bank

      Bank synchronization, reconciliation, and cash registers

   .. card:: Taxes
      :target: accounting/taxes

      Taxes, fiscal positions, and integrations

   .. card:: Reporting
      :target: accounting/reporting

      Reporting, declarations, and analytic accounting

Double-entry bookkeeping
========================

Odoo automatically creates all the behind-the-scenes journal entries

> for each of your accounting transactions: customer invoices, point of

Reviewer: please review this entire doc and remove all you/your mentions throughout

sale order, expenses, inventory moves, etc.

> Odoo uses the rules of double-entry bookkeeping system: all journal

Reviewer: ```suggestion
Odoo uses the rules of double-entry bookkeeping: all journal
```

entries are automatically balanced (sum of debits = sum of credits).

.. seealso::
   - :doc:`Understand Odoo's accounting transactions per document
     <accounting/getting_started/cheat_sheet>`

> Accrual and Cash Basis Methods

Reviewer: ```suggestion
Accrual and cash basis methods
```
follow proper capitalization guidelines for headers (only capitalize the first word)

==============================

> Odoo supports both accrual and cash basis reporting. This allows you to

Reviewer: considering that this seems like a repetitive error - please adhere the comment re: the removal of "you/your" to this entire PR. thanks

> report income / expense at the time transactions occur (i.e., accrual basis), or when

Reviewer: ```suggestion
report income / expense at the time transactions occur (i.e. accrual basis), or when
```

payment is made or received (i.e., cash basis).

Multi-companies
===============

Odoo allows one to manage several companies within the same database. Each
company has its own chart of accounts and rules. You can get
consolidation reports following your consolidation rules.

> Users can access several companies but always work in one company at a

Reviewer: ```suggestion
Users can access several companies, but always work in one company at a time
```
also seems like the writer didn't take full advantage of the 100th character/line cut-off - sentences, like this, are cut off far too soon/too abrupt. 

Please adhere this comment throughout the entirety of the doc

time.

Multi-currencies
================

Every transaction is recorded in the default currency of the
company. For transactions occurring in another currency, Odoo stores
both the value in the currency of the company and the value in the

> currency of the transaction. Odoo can generate currencies gains and

Reviewer: ```suggestion
currency of the transaction. Odoo can generate currency gains and
```

losses after the reconciliation of the journal items.

> Currency rates are updated once a day using a yahoo.com online

Reviewer: ```suggestion
Currency rates are updated once a day using a *Yahoo.com* online
```

web-service.

> International Standards

Reviewer: another repetitive error that needs to be addressed throughout the doc: proper header capitalization

Reviewer: ```suggestion
International standards
```

=======================

> Odoo accounting supports more than 50 countries. The Odoo core

Reviewer: be sure to italicize the name of the application/module (excluding the 'Odoo' preface), as follows.

```suggestion
Odoo *Accounting* supports more than 50 countries. The Odoo core
```
this is another comment that must be adhered to throughout the doc. Please review entire PR and make the necessary adjustments.

accounting implements accounting standards that are common to all
countries. Specific modules exist per country for the

> specificities of the country like the chart of accounts, taxes, or

Reviewer: ```suggestion
specificities of the country, like the chart of accounts, taxes, or
```
